server/resp/src/main/java/org/infinispan/server/resp/commands/set/SMISMEMBER.java

```diff
@@ -0,0 +1,35 @@
+package org.infinispan.server.resp.commands.set;
+
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.multimap.impl.EmbeddedSetCache;
+import org.infinispan.server.resp.Consumers;
+import org.infinispan.server.resp.Resp3Handler;
+import org.infinispan.server.resp.RespCommand;
+import org.infinispan.server.resp.RespRequestHandler;
+import org.infinispan.server.resp.commands.Resp3Command;
+
+import io.netty.channel.ChannelHandlerContext;
+
+/**
+ * {@link} https://redis.io/commands/smismember/
+ *
+ * @since 15.0
+ */
+public class SMISMEMBER extends RespCommand implements Resp3Command {
+   public SMISMEMBER() {
+      super(-3, 1, 1, 1);
+   }
+
+   @Override
+   public CompletionStage<RespRequestHandler> perform(Resp3Handler handler,
+         ChannelHandlerContext ctx,
+         List<byte[]> arguments) {
+      EmbeddedSetCache<byte[], byte[]> esc = handler.getEmbeddedSetCache();
+      var resultStage = esc.get(arguments.get(0))
+            .thenApply(s -> arguments.stream().skip(1)
+                  .map(i -> (s != null && s.contains(i) ? 1L : 0L)).toList());
```

Comment:
Good ol' non-atomic command, it would be possible for an entry to exist by the time the command finishes. We should be able to fix it with a `compute` function on the FunctionalMap. But I am good as it is, we can improve later. Maybe we add docs?

Reply:
I reworked SISMEMBER too and moved to atomic